tests/test_component.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+from mltrace.entities import components, base_component
```

Comment:
remove import to components since we are not actually having the component library yet

Reply:
removed